<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>550</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>714</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 249, 156);
font: 75 14pt &quot;Aharoni&quot;;
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>15</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>选择性别</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>40</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>男生</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>女生</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>541</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Aharoni&quot;;
background-color: rgb(162, 255, 237);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>420</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>合成</string>
    </property>
    <property name="icon">
     <iconset resource="leizi.qrc">
      <normaloff>:/hh/play48.png</normaloff>:/hh/play48.png</iconset>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>500</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 9pt &quot;Aharoni&quot;;</string>
    </property>
    <property name="text">
     <string>本人软件归作者所有</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 9pt &quot;Aharoni&quot;;
font: 75 10pt &quot;Aharoni&quot;;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>714</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionDakai"/>
    <addaction name="actionTuichu"/>
    <addaction name="actionQingkong"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="actionJianqie"/>
    <addaction name="actionFuzhi"/>
    <addaction name="actionZhantie"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="action"/>
    <addaction name="actionZuozhe"/>
    <addaction name="actionRuanjian"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <action name="actionDakai">
   <property name="icon">
    <iconset resource="leizi.qrc">
     <normaloff>:/hh/open.png</normaloff>:/hh/open.png</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionTuichu">
   <property name="icon">
    <iconset resource="leizi.qrc">
     <normaloff>:/hh/f08b.png</normaloff>:/hh/f08b.png</iconset>
   </property>
   <property name="iconText">
    <string>退出</string>
   </property>
  </action>
  <action name="actionQingkong">
   <property name="icon">
    <iconset resource="leizi.qrc">
     <normaloff>:/hh/1173189.gif</normaloff>:/hh/1173189.gif</iconset>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </action>
  <action name="actionJianqie">
   <property name="icon">
    <iconset resource="leizi.qrc">
     <normaloff>:/hh/cut.png</normaloff>:/hh/cut.png</iconset>
   </property>
   <property name="text">
    <string>剪切</string>
   </property>
  </action>
  <action name="actionFuzhi">
   <property name="icon">
    <iconset resource="leizi.qrc">
     <normaloff>:/hh/copy_clipboard_lined.png</normaloff>:/hh/copy_clipboard_lined.png</iconset>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
  </action>
  <action name="actionZhantie">
   <property name="icon">
    <iconset resource="leizi.qrc">
     <normaloff>:/hh/order.png</normaloff>:/hh/order.png</iconset>
   </property>
   <property name="text">
    <string>粘贴</string>
   </property>
  </action>
  <action name="action">
   <property name="icon">
    <iconset resource="leizi.qrc">
     <normaloff>:/hh/male-user-accept.png</normaloff>:/hh/male-user-accept.png</iconset>
   </property>
   <property name="text">
    <string>用户</string>
   </property>
  </action>
  <action name="actionZuozhe">
   <property name="icon">
    <iconset resource="leizi.qrc">
     <normaloff>:/hh/about-me.png</normaloff>:/hh/about-me.png</iconset>
   </property>
   <property name="text">
    <string>作者</string>
   </property>
  </action>
  <action name="actionRuanjian">
   <property name="icon">
    <iconset resource="leizi.qrc">
     <normaloff>:/hh/1170244.gif</normaloff>:/hh/1170244.gif</iconset>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="leizi.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>457</y>
    </hint>
    <hint type="destinationlabel">
     <x>475</x>
     <y>453</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
